contracts/test/spaces/roles/Roles.t.sol

```diff
@@ -1016,4 +1029,91 @@ contract RolesTest is BaseSetup, IRolesBase, IEntitlementBase {
     vm.expectRevert(Roles__EntitlementDoesNotExist.selector);
     roles.removeRoleFromEntitlement(roleId, entitlement);
   }
+
+  // =============================================================
+  // Channel Permissions
+  // =============================================================
+  function test_createCustomChannelPermissions()
+    external
+    givenRoleExists
+    givenRoleIsInChannel
```

Comment:
Are we always assuming the role is in the channel before anyone calls the createCustomChannelPermissions method?